Source/AhriAndBear/ABAnimalCharacter.h

```diff
@@ -32,20 +32,38 @@ class AHRIANDBEAR_API AABAnimalCharacter : public ACharacter
 		class UCameraComponent* camera;
 	UPROPERTY(EditAnywhere, BlueprintReadOnly)
 		UBoxComponent* InterationTrigger;
+	UPROPERTY(VisibleDefaultsOnly, BlueprintReadWrite)
+	class USphereComponent* ProjectileStart;
 	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Survival")
 		UAABSurvivalComponent* SurvivalComponent;
 	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "AI")
 		UBehaviorTree* BehaviorTree;
-
+	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Gameplay")
+	float JumpingSpeed = 700.f;
+	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Gameplay")
+	float MinDepth = -100.f;
+	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Gameplay")
+	float MaxDepth = -500.f;
+	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Gameplay")
+	float MinHeight = 50.f;
+	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Gameplay")
+	float MaxHeight = 500.f;
+	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Gameplay")
+	float EdgeForwardOffset = 50.f;
```

Comment:
If these are related to jumping, might I suggest changing `Category` to `Gameplay | Jumping` and maybe adding jumping to the variable names, i.e. `MinJumpDepth`, `MaxJumpHeight`, etc?